src/ethsign/ethsign.go

```diff
@@ -649,24 +649,16 @@ func main() {
       Name: "import",
       Usage: "import hexadecimal private key into keystore",
       Flags: []cli.Flag{
-        cli.StringFlag{
-          Name:  "keystore",
+        cli.StringSliceFlag{
+          Name:  "key-store",
           Usage: "path to keystore",
+          EnvVar: "ETH_KEYSTORE",
+          Value: &defaultKeyStores,
         },
       },
       Action: func(c *cli.Context) error {
-        requireds := []string{
-          "keystore",
-        }
-
-        for _, required := range requireds {
-          if c.String(required) == "" {
-            return cli.NewExitError("ethsign: missing required parameter --"+required, 1)
-          }
-        }
-
         ks := keystore.NewKeyStore(
-          c.String("keystore"),
+          c.String("key-store"),
```

Comment:
So this is broken. Since the type of the arg changed to a slice, `c.String('key-store")` produces something that looks like `[....]`, so we end up including `[` in the path where we create the keystore.

I guess we need to have some logic that iterates over the `key-store` slice and picks the first path that exists? Or maybe we just need to keep it as a string and remove the default value of `defaultKeyStores`, I'm not really sure if having multiple possible key stores makes sense in this context?

Reply:
In the second case I guess we would fail if `ETH_KEYSTORE` is not set?